### PR TITLE
Displaying message issue number 21.

### PR DESCRIPTION
**Pull request by laxmankhanal**

### CODE REVIEW

File: Vaccination/View/File.swift [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//
+//  File.swift
+//  Vaccination
+//
+//  Created by leapfrog-laxman on 4/11/16.
+//  Copyright © 2016 Leapfrog. All rights reserved.
+//
```
Comment (sanjaymhj) [inline Vaccination/View/File.swift]: File Name and delete the comment

File: Vaccination/View/KeyBoardToolBar.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        return (actionOnDoneButton, actionOnCancelButton)
+    }
+
+}
+
+enum KeyBoardRequester {
+    case CheckButton
+    case CustomNotificatinButton
+    case DateOfBirthPicker
+}
```
Comment (sanjaymhj) [inline Vaccination/View/KeyBoardToolBar.swift]: line at EOF

File: Vaccination/View/VaccinationCell.swift [hunk 3]
```diff
@@ -16,6 +16,8 @@ class VaccinationCell: UITableViewCell {
     var delegate: VaccinationCellDelegate?
 
     @IBAction func vaccineTaken(sender: UIButton) {
-        delegate?.cellForCheckBoxClicked(self)
+        self.delegate?.cellForCheckBoxClicked(self)
+        print("happy")
```
Comment (sanjaymhj) [inline Vaccination/View/VaccinationCell.swift]: print("happy") ???

File: Vaccination/ViewController/MainVC.swift [hunk 4]
```diff
@@ -91,6 +100,22 @@ class MainVC: UIViewController {
         self.view.resignFirstResponder()
     }
 
+    func getStartedView() {
+        firstView = UIView(frame: CGRect(x: 0, y: 20, width: self.view.frame.width, height: self.view.frame.height))
+        startButton = UIButton(frame: CGRect(x: 0, y:self.view.frame.height - 80, width: self.view.frame.width / 2, height: 40))
```
Comment (sanjaymhj) [inline Vaccination/ViewController/MainVC.swift]: height - 80 
what about larger size devices?

File: Vaccination/View/File.swift (hunk 1 above)
Comment (laxmankhanal) [inline Vaccination/View/File.swift]: :ok:

File: Vaccination/View/KeyBoardToolBar.swift (hunk 2 above)
Comment (laxmankhanal) [inline Vaccination/View/KeyBoardToolBar.swift]: 🆗

File: Vaccination/View/VaccinationCell.swift (hunk 3 above)
Comment (laxmankhanal) [inline Vaccination/View/VaccinationCell.swift]: 🆗

File: Vaccination/ViewController/MainVC.swift (hunk 4 above)
Comment (laxmankhanal) [inline Vaccination/ViewController/MainVC.swift]: Rest of the part is handled by autolayout.

**Comment by sanjaymhj** [comment]
:+1:

**Comment by sanjaymhj** [comment]
:+1:

**Pull request merged**